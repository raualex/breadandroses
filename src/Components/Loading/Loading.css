.fist2 {
  animation-name: fist-pulse;
  animation-duration: 0.5s;
  animation-delay: 0s;
  animation-direction: alternate;
  animation-timing-function: ease-out;
  animation-iteration-count: infinite;
  animation-play-state: running;
  height: 200px;
  left: 200px;
  position: absolute;
  top: 240px;
  width: 200px;
  z-index: 4;
}

@keyframes fist-pulse {
  0% {
    left: 250px;
    position: absolute;
    top: 290px;
    height: 100px;
    width: 100px;
  }

  100% {
    left: 200px;
    position: absolute;
    top: 240px;
    height: 200px;
    width: 200px;
  }
}

.star2 {
  animation-name: pulse;
  animation-duration: 0.5s;
  animation-delay: 0s;
  animation-direction: alternate;
  animation-timing-function: ease-out;
  animation-iteration-count: infinite;
  animation-play-state: running;
  position: absolute;
  z-index: 3;
}

.star-container2 {
  align-items: center;
  display: flex;
  height: 571px;
  justify-content: center;
  left: 28%;
  position: absolute;
  top: 40%;
  width: 600px;
}

@keyframes pulse {
  0% {
    height: 371px;
    width: 400px;
  }

  100% {
    height: 571px;
    width: 600px;
  }
}

@-webkit-keyframes rotate1 {
    from { -webkit-transform: rotate(0deg); }
    to   { -webkit-transform: rotate(20deg); }
}
@-moz-keyframes rotate1 {
    from { -moz-transform: rotate(0deg); }
    to   { -moz-transform: rotate(20deg); }
}
