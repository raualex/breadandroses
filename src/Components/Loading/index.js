import React from 'react';
import star from '../../Utils/Assets/Images/back_star.png';
import fist from '../../Utils/Assets/Images/power.svg';
import './Loading.css';

const Loading = () => {

  return (
    <div className="sun-container2">
      <div className="star-container2">
        <img src={star} className="star2" alt="star" />
        <img src={fist} className="fist2" alt="fist" />
      </div>
    </div>
  )
}

export default Loading;