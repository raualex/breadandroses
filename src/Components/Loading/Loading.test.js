import React from 'react';
import Loading from './';
import { shallow } from 'enzyme';

describe('Loading', () => {

  it('should match the snapshot', () => {
    
    let wrapper = shallow(<Loading />)

    expect(wrapper).toMatchSnapshot()
  });
});