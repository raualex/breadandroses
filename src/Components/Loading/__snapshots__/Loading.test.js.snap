// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="star-container2"
      >
        <img
          alt="star"
          className="star2"
          src="back_star.png"
        />
        <img
          alt="fist"
          className="fist2"
          src="power.svg"
        />
      </div>,
      "className": "sun-container2",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="star"
            className="star2"
            src="back_star.png"
          />,
          <img
            alt="fist"
            className="fist2"
            src="power.svg"
          />,
        ],
        "className": "star-container2",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "star",
            "className": "star2",
            "src": "back_star.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "fist",
            "className": "fist2",
            "src": "power.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="star-container2"
        >
          <img
            alt="star"
            className="star2"
            src="back_star.png"
          />
          <img
            alt="fist"
            className="fist2"
            src="power.svg"
          />
        </div>,
        "className": "sun-container2",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="star"
              className="star2"
              src="back_star.png"
            />,
            <img
              alt="fist"
              className="fist2"
              src="power.svg"
            />,
          ],
          "className": "star-container2",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "star",
              "className": "star2",
              "src": "back_star.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "fist",
              "className": "fist2",
              "src": "power.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
